src/scss/06_components/paragraphs/_hero.scss

```diff
@@ -70,6 +70,30 @@ $koro-height: 50px;
   }
 }
 
+[dir='rtl'] .hero__arrow {
+  left: auto;
+  right: $spacing-and-half;
+
+  @include breakpoint($breakpoint-l) {
+    right: $spacing-double;
+  }
+}
+
+// Hero image author styles.
+.hero__image-author {
+  // Arrow element width.
+  margin-left: 100px;
```

Comment:
Instead of a comment I'd put this value into a descriptive css variable (custom property) like `--arrow-element-width: 100px`, so that I could use it both in the regular and in the [dir="rtl"] cases.